src/basic/xfork.h

```diff
@@ -0,0 +1,35 @@
+#pragma once
+
+#include <sched.h>
```

Comment:
@karcherm Can you add the copyright stuff so we have one thing less that keeps this from being merged :-).